cluster_pack/spark/spark_config_builder.py

```diff
@@ -0,0 +1,39 @@
+
+import os
+
+import pyspark
+from pyspark.sql import SparkSession
+
+from cluster_pack import packaging, uploader
+
+from typing import Optional
+
+
+def add_script(ssb: SparkSession.Builder, archive: str):

```

Comment:
add_script doesn't really reflect what the function does, could we use something like 'add_packaged_environment' instead ?

Reply:
Done